daemon/volumes.go

```diff
@@ -103,6 +103,9 @@ func (daemon *Daemon) registerMountPoints(container *container.Container, hostCo
 		}
 
 		for _, m := range c.MountPoints {
+			if _, ok := mountPoints[m.Destination]; ok {
+				return fmt.Errorf("Duplicate mount point '%s'", m.Destination)
```

Comment:
I think we'll need to pick which one wins and *log* an error for the other one.
And the one we pick is whatever happens to win right now.